apps/dashboard/app/views/files/turbo_frames/_files.html.erb

```diff
@@ -26,7 +26,9 @@
 <% files.each do |file| %>
   <tr>
     <td>
-      <i class="fa <%= file[:directory] ? 'fa-folder' : 'fa-file' %>">
```

Comment:
I don't know about the word `folder` - `directory` is typically used in Linux (folder being more of a Windows word).

I any case - we should internationalize both.